server/container/util/src/main/java/org/apache/james/util/OptionalUtils.java

```diff
@@ -42,6 +42,10 @@
             .orElse(Stream.of());
     }
 
+    public static <T> Stream<T> ofNullableToStream(T maybeNull) {
+        return toStream(Optional.ofNullable(maybeNull));
+    }
```

Comment:
missing test (and I don't really see the benefit)

Reply:
Well I do find `OptionalUtils.toStream(Optional.ofNullable(X)` boiler plate. 

I will add the tests.

Reply:
maybe calling it streamOfNullable (similar of Optional.ofNullable) could be better then

Reply:
Crap, we can use `StreamUtils.ofNullable(X)` instead, I will do the replacement...

Reply:
you can remove this now

Reply:
cf https://github.com/linagora/james-project/pull/1702/commits/d8d06a7c07188a92274199e921aebfff646b4f4c

Reply:
exactly, in this commit you missed to remove the method you added (you just remove the tests) ^^